paprika_recipes/remote.py

```diff
@@ -75,6 +76,30 @@ def get_recipe_by_id(self, id: str, hash: str) -> RemoteRecipe:
     def count(self) -> int:
         return len(self._get_remote_recipe_identifiers())
 
+    def download_image(self, recipe: RemoteRecipe):
```

Comment:
Really, the `Remote` object probably should probably focus on just providing access to data so that other callers (e.g. the `download-recipes` command) can decide what to do with that data.  Also, this'll create a new folder named `images` in your current working directory instead of at your export path; so there are a few things I'd probably recommend changing:

1. Rename this function something like `fetch_recipe_image`
2. Have it return the response data (just `return result.content`, I think?)
3. Make the caller figure out where to write the image to (it's the only place that knows where you want to write this)